dex/testing/dcrdex/harness.sh

```diff
@@ -217,14 +217,19 @@ EOF
 fi
 
 if [ $ETH_ON -eq 0 ]; then
+
+ETH_CONFIG_PATH=${HARNESS_DIR}/eth.conf
+cat << EOF >> $ETH_CONFIG_PATH
+ws://localhost:38557
+EOF
```

Comment:
Since the markets.json in dcrdex/harness.sh tends to be demonstrative, would you mind adding a `#`-prefixed comment with a second provider as an example?  Also, could we actually have the harness use two nodes so it's more than an example?